_projects/2022/100385586/100385586.Rmd

```diff
@@ -0,0 +1,310 @@
+
+
+## Set up and libraries 
+
+As with any other project, we have to load some the libraries that will be needed to complete the graph.
+
+```{r setup, fig.showtext=TRUE, echo=TRUE, message=FALSE, warning=FALSE}
+library(tidyverse)
+library(gganimate)
+library(gifski)
+library(plotly)
+library(ggtext)
+library(RColorBrewer)
+library(htmlwidgets)
+library(ggtext)
+library(showtext)
+
+```
+
+## Prerequisites for the graph
+
+### Adapting the database. 
+
+First, we have to load the data. In this case, it has been directly compiled from the Pew Research Center. 
+
+Some columns are going to be renamed and/or modified to make easier the coding. Also, a new variable is created to give later the colour to the bubbles (geom_point()) via the case_when() function. 
+
+```{r}
+religion.df <- read_delim("rel.csv", show_col_types = FALSE)
+
+religion <- religion.df %>% transmute(
+  year = as.integer(`Year `), 
+  GRI = GRI, 
+  SHI = SHI,
+  Country = Country, 
+  Pop = `Population Size `) %>% 
+  mutate(
+  pos = case_when( # The position is numerically defined according to the score in the variables
+    (GRI < 2.3 & SHI < 1.4) ~ 1, 
+    ((GRI >= 2.3 & GRI < 4.5) & SHI < 1.4) ~ 2, 
+    ((GRI >= 4.5 & GRI < 6.6) & SHI < 1.4) ~ 3, 
+    ((GRI >= 6.6 & GRI < 10)  & SHI < 1.4) ~ 4,
+    ((GRI >= 0 & GRI < 2.3)   & SHI < 3.6) ~ 2,
+    ((GRI >= 2.3 & GRI < 4.5) & SHI < 3.6) ~ 3, 
+    ((GRI >= 4.5 & GRI < 6.6) & SHI < 3.6) ~ 4, 
+    ((GRI >= 6.6 & GRI < 10)  & SHI < 3.6) ~ 5, 
+    ((GRI >= 0 & GRI < 2.3)   & SHI < 7.2) ~ 3,
+    ((GRI >= 2.3 & GRI < 4.5) & SHI < 7.2) ~ 4, 
+    ((GRI >= 4.5 & GRI < 6.6) & SHI < 7.2) ~ 5, 
+    ((GRI >= 6.6 & GRI < 10)  & SHI < 7.2) ~ 6,
+    ((GRI >= 0 & GRI < 2.3)   & SHI <= 10) ~ 4,
+    ((GRI >= 2.3 & GRI < 4.5) & SHI <= 10) ~ 5, 
+    ((GRI >= 4.5 & GRI < 6.6) & SHI <= 10) ~ 6, 
+    ((GRI >= 6.6 & GRI < 10)  & SHI <= 10) ~ 7)) %>% 
+  mutate(
+    pos = as.factor(pos), 
+    Country = as.factor(Country))
+  
+```
+
+### Elements and labels. 
+
+At this preliminary stage, some elements of the graph, such as the breaks, titles and subtitles are defined for the same purpose of making the code easier and more understandable. 
+
+#### Text elements 
+
+```{r}
+title <- paste ("Restrictions on religion among the 25 most populous countries,",  
+                "2007-2020", sep = "\n")
+
+subtitle  <- paste ( 
+"Among the 25 most populous countries, India, Egypt, Pakistan, Indonesia and Nigeria had the highest levels of overall restrictions on", 
+"religion in 2020, while Japan, the United States, Italy, the Democratic Republic of the Congo and Tanzania had the fewest restrictions.", 
+"Click play to see how restrictions have changed in each country since 2007. Read the full report.", 
+"Year: {closest_state}", 
+sep = "\n") # The phrases have to be written independently and separated with the \n function. 
```

Comment:
If you see [the preview](https://deploy-preview-9--mucss-dataviz.netlify.app/_projects/2022/100385586/100385586), this text takes too much width and jumps out of the screen. Please, split it in shorter strings.